packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue

```diff
@@ -31,6 +31,12 @@
                 @keydown.down="setKeyboardSuggestion(1)">
 
             <span :class="$style['c-search-placeholder']">{{ copy.fieldPlaceholder }}</span>
+
+            <div
```

Comment:
I might be remembering wrong, but I thought you had separated the loading spinner into its own component, could we use that here/did I make that up? 😂 🤔

Reply:
I think maybe it was the SCSS styles I moved to fozzie so it was shareable? at the moment this is pulling from there.

Reply:
I think you're right - that's what I'm thinking of 👍